### PR TITLE
Reinstate support for Ubuntu Focal

### LINKED ISSUE #206: Consider adding support for Ubuntu Focal
Ubuntu 20.04 is still supported until April 2025, so we should consider adding it to the list of supported releases for users who haven't moved to Jammy yet.

### PR DESCRIPTION
**Pull request by apyrgio**
Reinstate support for Ubuntu Focal, by creating packages for it in our CI and providing some extra installation instructions to users.

Closes #206
Refs #240

### CODE REVIEW

**Review by deeplow — COMMENTED** [review]
Some other stuff that needs to be done:
- [ ] Change some [website links](https://github.com/freedomofpress/dangerzone.rocks/blob/9699fea2d67a339f3e6cf3d9e39136a2d694c54b/index.html#L96)
- [ ] Add a "redirect" link in the wiki

Other than that it looks good. I'll test this on ubuntu focal, just in case.

File: install/podman.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+if [[ "$1" == "--repo-only" ]]; then
+    provide
+elif [[ "$1" == "" ]]; then
+    provide
+    install
+else
+    echo "Unexpected argument: $1"
+    echo "Usage: ./install/podman.sh [--repo-only]"
```
[the remark this answers is not in the document]
Comment (apyrgio) [reply install/podman.sh]: I agree, I fixed it as well.

**Comment by apyrgio** [comment]
Thanks for your suggestions. I incorporated them, so I'm good to merge when I have the ACK from you.

**Comment by deeplow** [comment]
LGTM

**Pull request merged**